hw7.py

```diff
@@ -0,0 +1,26 @@
+phonebook = {}
+while True:
+    user_input = input('Enter command: ')   # format command name phone
+    split_input = user_input.split()
+    command = split_input[0]
+    name = split_input[1]
+    phone = split_input[2]
+    if phonebook.get(name) is None:

```

Comment:
This condition can be moved somewhere you work with keys. `stats` command, for example, won't be executed because of this check

Reply:
condition moved under "add" command